Stripe/STPPaymentContext.m

```diff
@@ -568,7 +568,7 @@ - (void)requestSourceFlowRedirectPayment:(STPSource *)source {
     STPSourceCompletionBlock onRedirectCompletion = ^(STPSource *finishedSource, NSError *error) {
         stpDispatchToMainThreadIfNecessary(^{
             if (error) {
-                [self didFinishWithStatus:STPPaymentStatusError error:error];
+                [self didFinishWithStatus:STPPaymentStatusPending error:nil];
```

Comment:
Hm, I think we should treat all polling errors equally here – the source has been created at this point, and the user may have authorized the payment, but we don't know because there was an error. Even if there was a connectivity error while polling, it's still possible the user authorized the payment before they went offline.